.github/workflows/tests.yml

```
@@ -24,7 +24,7 @@ jobs:
           west init -l .
           west update -o=--depth=1 -n
 
-      - name: Build firmware
+      - name: Twister Tests (app)
```

```suggestion
      - name: Build application
```
As this task is purely a build. No tests are run. Not even on simulated environment